pagination/src/main/java/com/cerner/beadledom/pagination/models/OffsetPaginatedListDtoImpl.java

```diff
@@ -0,0 +1,53 @@
+package com.cerner.beadledom.pagination.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.google.auto.value.AutoValue;
+import java.util.ArrayList;
+
+/**
+ * Offset based pagination list.
+ *
+ * <p>Represents an offset based paginated list in a rest response.
+ *
+ * @author John Leacox
+ * @since 3.1
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@AutoValue
+@JsonPropertyOrder({
+    "items",
+    "totalResults",
+    "firstLink",
+    "lastLink",
+    "nextLink",
+    "prevLink"
+})
+@JsonDeserialize(builder = OffsetPaginatedListDtoImpl.Builder.class)
+public abstract class OffsetPaginatedListDtoImpl<T> implements OffsetPaginatedListDto<T> {
```

Comment:
I didnt make this package private since this is in the `models` package and is consumed by `OffsetPaginatedListLinksWriterInterceptor` which is in root level package.

I can move to be in the same package but didnt know how others have felt.

Reply:
Yeah, go ahead and move it so it can be package-private.

Reply:
Moved the package https://github.com/cerner/beadledom/pull/116/commits/feea8d23c8cdb2ed01f230db08f3bcd579354fe4